openedx/frontend-app-publisher
# feat: add fields direction change option on course edit form

Description:
-------------
[PROD-3220] (https://2u-internal.atlassian.net/browse/PROD-3220)

- Demo is in Ticket description

Testing:
In order to test the changes on local
- Fetch backed Discovery discovery changes PR (to allow direction attr to add in p tag) https://github.com/openedx/course-discovery/pull/3908
- Run `discovery`, `frontend-app-publisher` and `studio` containers on local
- Go to edit course page
- Verify direction of `title` and `url_slug` changing on switching the switch button 
- Plus check directionality plugin working for `tinymce` editor

> Will add these changes to create course form as well, once i got review from the team 

##### Snapshot:
<img width="1347" alt="image" src="https://user-images.githubusercontent.com/78806673/236929727-aa6c1435-68b3-45d1-aeee-e2e702077e41.png">


Vedio Demo:

https://user-images.githubusercontent.com/78806673/234691580-c8098a24-bcbd-487e-89c6-ce7b3f36f4c7.mp4
